data/map.txt

```diff
@@ -24252,7 +24252,7 @@ planet "Brass Second"
 
 planet "Bright Echo"
 	attributes saryd tourism urban
-	landscape land/dmottl1
+	landscape land/mountain14-sfiera
```

Comment:
Given that the Saryd settlements are park-like, with buildings nestled among trees rather than filling large clear-cut areas, this particular image doesn't work very well for them.